dev-python/speg/speg-0.3.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A PEG-based parser interpreter with memoization"
+DESCRIPTION="A PEG-based parser interpreter with memoization (in time)."
+HOMEPAGE="https://github.com/avakar/speg"
+COPYRIGHT="2015-2017, Martin Vejnár"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/source/s/speg/speg-$portVersion.zip"
+CHECKSUM_SHA256="10cbef47e168dfc62f14db575cf1c428037a2b881cee6c3cfceda0439c243e71"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	speg = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_VERSIONS=(3.10)
+
+for i in "${!PYTHON_VERSIONS[@]}"; do
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+	pythonPackage=python${pythonVersion//.}
+
+	eval "PROVIDES_${pythonPackage}=\"
+		speg_$pythonPackage = $portVersion
+		\""
+	eval "REQUIRES_$pythonPackage=\"
+		$REQUIRES
+		cmd:python$pythonVersion
+		six_$pythonPackage
+		\""
+
+	BUILD_REQUIRES+="
+		setuptools_$pythonPackage
+		"
+	BUILD_PREREQUIRES+="
+		cmd:python$pythonVersion
+		"
+done
+
+INSTALL()
+{
+	for i in "${!PYTHON_VERSIONS[@]}"; do
+		pythonVersion=${PYTHON_VERSIONS[$i]}
+		pythonPackage=python${pythonVersion//.}
+
+		python=python$pythonVersion
+		installLocation=$prefix/lib/$python/vendor-packages/
+		export PYTHONPATH=$installLocation:$PYTHONPATH
+
+		mkdir -p $installLocation
+		rm -rf build
+
+		$python setup.py build install \
+			--root=/ --prefix=$prefix
```

Comment:
This can't be built with `build`. It gives an error message similar to "invalid command bdist_wheel".

Reply:
Thanks, with `wheel_$pythonPackage` it works.